src/Symfony/Component/Console/ConsoleEvents.php

```diff
@@ -27,6 +27,14 @@ final class ConsoleEvents
      */
     const COMMAND = 'console.command';
 
+    /**
+     * The STOP event allows you to perform some actions
+     * after the command execution was interrupted.
+     *
```

Comment:
Maybe add here an info that it requires the pcntl ext to work properly

Reply:
@noniagriconomie There is already a check in the Application class and we don't want to add an hard deps. so it's fine for me